### ISSUE TITLE
ImHex

### ISSUE THREAD

**Issue by DorielRivalet** [catppuccin/catppuccin#2049]
### Link to repository

https://github.com/DorielRivalet/imhex.git

### Screenshots

![preview](https://github.com/DorielRivalet/imhex/raw/main/assets/preview.webp)

### Any additional comments?

Tested on Windows 10

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by sgoudham** [comment]
Hiya :wave: 

Sorry, it took us a while to get around to this! I just tested it out there and it looks great! Thanks for your work on this :heart: 

I'd just like to add a couple of comments if that's alright:

1. It would make sense to also publish these JSON files under a GitHub release, just to ease the pain of actually having to clone the repository or download the individual files
2. In the [Manual Installation](https://github.com/DorielRivalet/imhex#manual-installation) section of the README, you mention the installation instructions for Windows but ImHex supports a variety of operating systems. I think it would be good to specify a set of installation instructions for Windows, macOS, and Linux. Or alternatively, we can refer the user to the official help (`Help` -> `About` -> `ImHexDirectories` -> `Themes`) to figure out what directories to install it in.
3. I've mentioned in other issues that I'd like to see the JSON files generated via our [palette libraries](https://github.com/catppuccin/palette) but it makes sense to raise this as an issue on the repository itself!

`1.` and `3.`  can be implemented at any time and don't relate to the quality of the port. I'm personally happy to commit an improvement to `2.` so I'm happy to approve this! I've sent you an invite to the organisation and feel free to transfer the repository!

**We'll close this issue once the port has been added to our [ports.yml](https://github.com/catppuccin/catppuccin/blob/main/resources/ports.yml)!**
